episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,847 @@
+  filter(day_max < 1) %>%
+  nrow()
+```
+
+Although most introductions of `r initial_cases` index case do not generate secondary cases (N = `r chains_null`) or most outbreaks rapidly become extinct (median duration of `r chains_extinct$extinct_duration_median` and median size of `r chains_extinct$extinct_size_median`), only `r threshhold_summary$chains_theshold` epidemic trajectories among `r threshhold_summary$chains_number` simulations (`r threshhold_summary$chains_percentage`%) can reach to more than 100 infected cases. This finding is particularly remarkable because the reproduction number $R$ is less than 1 (offspring distribution mean of `r mers_offspring[["mean"]]`), but, given an offspring distribution dispersion parameter of `r mers_offspring[["dispersion"]]`, it shows the potential for explosive outbreaks of MERS disease.
+
+::::::::::::::::::::::::::::::::::: spoiler
+
+### Observed cases vs simulated chains
+
+Let's overlap the cumulative number of observed cases using the linelist object from the `mers_korea_2015` dataset of the `{outbreaks}` R package. To prepare the dataset we use `{incidence2}` to convert the linelist to an `<incidence2>` object, complete the missing dates of the time series with `complete_dates()`
+
+```{r,warning=FALSE,message=FALSE}
+library(outbreaks)
+
+mers_cumcases <- mers_korea_2015$linelist %>%
+  # incidence2 workflow
+  incidence2::incidence(date_index = "dt_onset") %>%
+  incidence2::complete_dates() %>%
+  # wrangling using {dplyr}
+  mutate(count_cumsum = cumsum(count)) %>%
+  rownames_to_column(var = "day") %>%
+  mutate(day = as.numeric(day))
+```
+
+Use `plot()` to make an incidence plot:
+
+```{r}
+# plot the incidence2 object
+plot(mers_cumcases)
+```
+
+:::::::::::::::::::::::::::::::::::
+
+When plotting the observed number of cumulative cases from the Middle East respiratory syndrome (MERS) outbreak in South Korea in 2015 alongside the previously simulated chains, we identify that the observed cases followed a trajectory that resembled a simulated explosive outbreak.
+
+```{r,echo=FALSE}
+# Observed cases vs Simulated transmission chains
+mers_cumcases_type <-
+  mers_cumcases %>%
+  mutate(type = "Observed")
+simulated_chains_day_type <-
+  simulated_chains_day %>%
+  mutate(type = "Simulated")
+
+ggplot() +
+  geom_line(
+    data = simulated_chains_day_type,
+    mapping = aes(
+      x = day,
+      y = cases_cumsum,
+      group = chain_id,
+      color = type
+    )
+  ) +
+  geom_hline(aes(yintercept = 100), lty = 2) +
+  geom_line(
+    data = mers_cumcases_type,
+    mapping = aes(
+      x = day,
+      y = count_cumsum,
+      color = type
+    ),
+    linewidth = 1.5
+  ) +
+  labs(
+    x = "Day since first report",
+    y = "Cumulative cases",
+    color = "Type"
+  )
+```
+
+When we increase the dispersion parameter from $k = 0.01$ to $k = \infty$, given a fixed reproduction number $R = 1.5$, the proportion of simulated outbreaks that reached the 100-case threshold increases.
```

Comment:
```suggestion
When we increase the dispersion parameter from $k = 0.01$ to $k = \infty$ - and hence reduce individual-level variation in transmission - and assume a fixed reproduction number $R = 1.5$, the proportion of simulated outbreaks that reached the 100-case threshold increases. This is because the simulated outbreaks now have more of a consistent, clockwise dynamic, rather than the high level of variability seen previously.
```